src/webserver/Dynamic.jl

```diff
@@ -251,39 +251,11 @@ end"""
 end"""
 
     elseif extension ∈ ["jl"]
-        code = """include("$(path)")"""
-
-    elseif extension ∈ ["csv", "json"]
-        requirements = [:TableIO, :DataFrames]
-        req_code = "using TableIO, DataFrames"
-        code = """begin
-    $(req_code)
-    df_$(varname) = read_table("$(path)") |> DataFrame
-end"""
-
-    elseif extension ∈ ["arrow"]
-        requirements = [:TableIO, :DataFrames]
-        req_code = "using TableIO, DataFrames"
-        code = """begin
-    $(req_code)
-    df_$(varname) = DataFrame(read_table("$(path)"), copycols=false)
-end"""
-
-    elseif extension ∈ ["xlsx", "xslm"]
-        requirements = [:TableIO, :DataFrames]
-        req_code = "using TableIO, DataFrames"
-        code = """begin
-    $(req_code)
-    df_$(varname) = DataFrame(read_table("$(path)"#==,  "Sheet1" ==#); copycols=false)
-end"""
+        file = open(path)
+        code = read(file, String)
 
-    elseif extension ∈ ["db", "sqlite", "sqlite3"]
-        requirements = [:TableIO, :DataFrames]
-        req_code = "using TableIO, DataFrames"
-        code = """begin
-    $(req_code)
-    df_$(varname) = DataFrame(read_table("$(path)"), "table" #== Replace this with the table ==#, copycols=false)
-end"""
+    elseif extension ∈  ["zip",  "csv",  "jdf",  "parquet",  "xlsx",  "xlsm",  "db",  "sqlite",  "sqlite3",  "dta",  "sav",  "sas7bdat",  "json",  "arrow"]
+        code = get_example_code(directory, filename)
```

Comment:
it would be good if the supported file extensions are not needed to be defined in Pluto.
I could add a function in TableIOInterface to check if an extension is supported.

Reply:
I added the function `is_extension_supported(extension:: AbstractString)` in TableIOInterface.jl